### ISSUE TITLE
Setting gallery with pixel-based width and pixel-based gutter

### ISSUE THREAD

**Issue by patrickbjohnson** [ericam/susy#520]
I'm unable to find a way to create a gallery grid with a fixed width and column, but scales as well. Here's a paired down test: http://sassmeister.com/gist/99ff4cce02003941b783

Not sure if I'm doing something incorrectly? I want the max-width of each `portfolio-item` should be `360px` with a `30px` gutter.

**Comment by ericam** [comment]
I believe you want [something like this](http://sassmeister.com/gist/1b92d9057531cddcb8c9)?

**Comment by patrickbjohnson** [comment]
This would be it! Didn't know I'd need to establish a `$susy` map to set up specific column widths. 

Will doing this here effect any other reference to `@include gallery()` before or after these lines of code in my CSS?

**Comment by ericam** [comment]
Well, that setting really only affects the container - where we set `max-width`. But yes, anything you change in the global `$susy` map will affect everything. You can also create your own alternative settings maps and pass them in wherever you want:

```scss
$this-gallery: (
  columns: 3,
  gutters: 30px/360px,
  column-width: 360px,
);

.body {
  @include container($this-gallery);
}

.portfolio-item{
  @include gallery(1 of $this-gallery);
}
```

**Comment by patrickbjohnson** [comment]
Hmm. This does work, however I'm now looking to establish the max-width of the container. Here's what I got (sorry for the hand holding)

```
$standout-container: (
  container: 1170px,
  columns: 3,
  gutters: 30px/360px,
  column-width: 360px,
);

.bar {
	@include container($standout-container);
}

.standout__block {
	@include gallery(1 of $standout-container);
}
```

Now, I imagine its because the math doesn't work out? But not sure if we can pass so many fixed values while still managing a fluid, percentage-based layout?

**Comment by ericam** [comment]
I'm not sure what error you are getting, if any, but it's not a great idea to set both `container` and `column-width`. The default setting for `container` is `auto`, chich will calculate the max-width based on `column-width`, `columns`, and `gutters`.

**Comment by patrickbjohnson** [comment]
That's what I figured. I wasn't getting an error just an undesired outcome. 

In any instance in which I set both `container` and `column-width` I ended up with an inaccurate `column-width`. It seems like in that case, where one sets both `column` and `container`, the `column` set is overridden based on the max-width of the container.

In this case I've reverted to using only the `column-width` and removed any reference to the container.
